blocks/amp-story/text/edit.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment } from '@wordpress/element';
+import { PanelBody, SelectControl, withFallbackStyles } from '@wordpress/components';
+import { RichText, InspectorControls, FontSizePicker, withFontSizes } from '@wordpress/editor';
+import { compose } from '@wordpress/compose';
+
+const { getComputedStyle } = window;
+
+/**
+ * Internal dependencies
+ */
+import getTagName from './getTagName';
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { fontSize, customFontSize } = ownProps.attributes;
+	const editableNode = node.querySelector( '[contenteditable="true"]' );
+	//verify if editableNode is available, before using getComputedStyle.
+	const computedStyles = editableNode ? getComputedStyle( editableNode ) : null;
+
+	return {
+		fallbackFontSize: fontSize || customFontSize || ! computedStyles ? undefined : parseInt( computedStyles.fontSize ) || undefined
```

Comment:
Do we need to resort to getting the computed styles? I'm concerned about the performance impact.

Reply:
FWIW I adapted this from the `core/paragraph` block, but haven't tested it properly yet.

`getComputedStyle` is meant as the fallback of the fallback.

Since the paragraph block is quite special in Gutenberg, `getComputedStyle` and the `editableNode` check might not even apply to our custom block here.